dev/projection/shared-relative-new-child.html

```diff
@@ -89,19 +89,18 @@
             newBoxProjection.root.didUpdate()
 
             sync.postRender(() => {
-                matchViewportBox(newBox, {
-                    bottom: 400,
-                    left: 100,
-                    right: 400,
-                    top: 150,
-                })
-
-                matchViewportBox(child, {
-                    bottom: 210,
-                    left: 340,
-                    right: 390,
-                    top: 160,
-                })
+                // matchViewportBox(newBox, {
```

Comment:
Why are we skipping this test? Is it no longer the expected behaviour?

Reply:
Ooooops thanks

Reply:
Please still pass

Reply:
NOOOO